### ISSUE TITLE
example from readme results in following error

### ISSUE THREAD

**Issue by yewalenikhil65** [rmsrosa/ChainPlot.jl#1]
https://github.com/rmsrosa/ChainPlot.jl#examples
with julia 1.7.1
```julia

julia> plot(nnr, title="With theme solarized_light", titlefontsize=10)
ERROR: MethodError: no method matching add_edge!(::MetaGraphs.MetaGraph{Int64, Float64}, ::Int64, ::Int64)
Closest candidates are:
  add_edge!(::LightGraphs.SimpleGraphs.AbstractSimpleGraph, ::Any, ::Any) at ~/.julia/packages/LightGraphs/IgJif/src/SimpleGraphs/SimpleGraphs.jl:90
  add_edge!(::LightGraphs.SimpleGraphs.AbstractSimpleGraph, ::Any) at ~/.julia/packages/LightGraphs/IgJif/src/SimpleGraphs/SimpleGraphs.jl:86
Stacktrace:
 [1] chaingraph(m::Chain{Tuple{Dense{typeof(σ), Matrix{Float32}, Vector{Float32}}, Flux.Recur{Flux.RNNCell{typeof(relu), Matrix{Float32}, Vector{Float32}, Matrix{Float32}}, Matrix{Float32}}, Flux.Recur{Flux.LSTMCell{Matrix{Float32}, Vector{Float32}, Tuple{Matrix{Float32}, Matrix{Float32}}}, Tuple{Matrix{Float32}, Matrix{Float32}}}, Flux.Recur{Flux.GRUCell{Matrix{Float32}, Vector{Float32}, Matrix{Float32}}, Matrix{Float32}}, Dense{typeof(identity), Matrix{Float32}, Vector{Float32}}}}, input_data::Nothing)
   @ ChainPlot ~/.julia/packages/ChainPlot/Ua2jN/src/chaingraph.jl:77
 [2] macro expansion
   @ ~/.julia/packages/ChainPlot/Ua2jN/src/chainplotrecipe.jl:56 [inlined]
 [3] apply_recipe(plotattributes::AbstractDict{Symbol, Any}, m::Chain, input_data::Union{Nothing, Array})
   @ ChainPlot ~/.julia/packages/RecipesBase/qpxEX/src/RecipesBase.jl:289
 [4] apply_recipe(plotattributes::AbstractDict{Symbol, Any}, m::Chain)
   @ ChainPlot ~/.julia/packages/RecipesBase/qpxEX/src/RecipesBase.jl:283
 [5] _process_userrecipes!(plt::Any, plotattributes::Any, args::Any)
   @ RecipesPipeline ~/.julia/packages/RecipesPipeline/oVorB/src/user_recipe.jl:36
 [6] recipe_pipeline!(plt::Any, plotattributes::Any, args::Any)
   @ RecipesPipeline ~/.julia/packages/RecipesPipeline/oVorB/src/RecipesPipeline.jl:70
 [7] _plot!(plt::Plots.Plot, plotattributes::Any, args::Any)
   @ Plots ~/.julia/packages/Plots/cPJQu/src/plot.jl:208
 [8] #plot#135
   @ ~/.julia/packages/Plots/cPJQu/src/plot.jl:91 [inlined]
 [9] top-level scope
```

**Comment by rmsrosa** [comment]
Yeah, LightGraphs.jl has been deprecated and MetaGraphs.jl now uses Graphs.jl and probably has a different API. I should work on updating this. Thanks for reporting!

**Comment by rmsrosa** [comment]
Ok, package updated to use `Graphs.jl`.

I took the opportunity to merge an improved way to find the layer connections, but which might have problems with RNN due to https://github.com/FluxML/Flux.jl/pull/1755

closed with commit 53b6d3b7d19b5632801eba0795e485d86f41a0e7
